tests/functional/test_rose_stem.py

```diff
@@ -489,49 +339,17 @@ def with_config(
         '[rose-stem]\n'
         'automatic-options=MILK=true\n'
     )
-    monkeymodule.setenv(
+    monkeypatch.setenv(
         'ROSE_CONF_PATH', str(setup_stem_repo['basetemp'])
     )
-    yield rose_stem_run_template(rose_stem_opts)
-    monkeymodule.delenv('ROSE_CONF_PATH')
```

Comment:
Monkeypatch/module should do this by itself?

Reply:
At the end of the module. I think we want to remove this now.